markdown/ml/ml8_testing/index.md

```diff
@@ -4,25 +4,76 @@ title: "NN8 - Performanzanalyse"
 menuTitle: "NN8 - Performanzanalyse"
 author: "Canan Yıldız (Türkisch-Deutsche Universität)"
 weight: 8
-outcomes: false
+outcomes: true
 # readings:
 #   - key: "Russell2020"
 #     comment: "Kapitel 2 und 3"
 #   - key: "Ertel2017"
 # quizzes:
 #   - link: XYZ
 #     name: "Testquizz (URL from 'Invite more Players')"
-# assignments:
-#   - topic: blatt01
-#   - topic: blatt02
-# youtube:
-#   - id: XYZ (ID)
-#   - id: XYZ (ID)
-sketch: true
+youtube:
+    - id: T-WYL28iwdU
+    - id: fpsNzn4Moow
+    - id: Wx_HAuIXTAQ
+    - id: UAV7EpdIe6Q
+    - id: vsmoYiArtrA
+sketch: false
 ---
 
 
-<div class="sketch" style="text-align:center;">
-Inhalt befindet sich im Aufbau<br>
-und wird rechtzeitig bereitgetellt.
-</div>
+## Kurze Übersicht
+
+### Performanzmetriken für Klassifizierungsprobleme
+
+#### Wahrheitsmatrix (engl. Confusion Matrix)
+*   Gibt eine Übersicht über die Anzahl von richtig und falsch klassifizierten Datenpunkten (bei binärer Klassifizierung)
+    *   $TP = $ # True Positives $ = $ Anzahl richtiger 1-Vorhersagen
+    *   $FP = $ # False Positives $ = $ Anzahl falscher 1-Vorhersagen
+    *   $FN = $ # False Negatives $ = $ Anzahl falscher 0-Vorhersagen
+    *   $TN = $ # True Negatives $ = $ Anzahl richtiger 0-Vorhersagen
+*   Bei Klassifizierungsproblemen mit $N$ Klassen hat man eine $N \times N$ Matrix, die in Position $(i,j)$ die Anzahl der Klasse-$j$-Beispiele enthält, die als Klasse-$i$ vorhergesagt wurden.
+
+![Abbildung 1 - Wahrheitsmatrix bei binärer Klassifizierung](images/nn8-1.png)
+
+#### Treffergenauigkeit (engl. Accuracy)
+*   Anzahl richtig klassifizierter Datenpunkte, Erfolgsrate (engl. correct rate)
+    $$ Accuracy = \frac{TP+TN}{TP+TN+FP+FN} $$
+
+*   Accuracy vermittelt ein falsches Bild des Erfolges bei unausgewogenen Datensätzen \
+    Beispiel: 
+    *   Klasse 1 hat 10, Klasse 0 hat 990 Beispiele.
+    *   Ein Modell, das immer 0 ausgibt, hat $990/1000 = 0.99$ Treffergenauigkeit, ist aber offensichtlich kein gutes Modell!
+
+#### Prezision (engl. Precision)
```

Comment:
```suggestion
#### Precision (engl. Precision)
```

Reply:
mist, den einen hatte ich übersehen ...

Reply:
Das Englische brauche ich in dem Fall nicht in Klammern anzugeben, oder?